.github/workflows/pull-request.yml

```diff
@@ -228,10 +228,12 @@ jobs:
         run: node ./script/github-actions/filter-files-changed.js ${{ steps.get-files-changed.outputs.files }}
 
       - name: Annotate ESLint results
-        run: yarn run eslint --ext .js --ext .jsx --no-error-on-unmatched-pattern --format ./script/github-actions/eslint-annotation-format.js ${{ steps.get-lint-files.outputs.FILES }}
+        run: yarn run eslint --ext .js --ext .jsx --no-error-on-unmatched-pattern --format ./script/github-actions/eslint-annotation-format.js ${{ steps.get-lint-files.outputs.JSFILES }}
 
-      - name: Run Sass lint
-        run: yarn run sass-lint --format json --verbose --output sasslint-report.json
+      - name: Run Stylelint
+        if: ${{ steps.get-lint-files.outputs.SCSSFILES != '' }}
+        run: yarn run stylelint verbose --output-file stylelint-report.json --formatter json ${{ steps.get-lint-files.outputs.SCSSFILES }}
 
-      - name: Annotate Sass lint results
-        run: node ./script/github-actions/sasslint-annotation-format.js
+      - name: Annotate Stylelint results
+        if: ${{ always() }}
```

Comment:
What is this doing?

Reply:
In order for the annotation to display accordingly in the correct file, it has to be outputted in a certain way. The step before it getting the output of error messages and then running this script to annotate according to github action `::error filename,line,column`

The `if` statement is guaranteeing if there is a lint that causes `Error`, it will still displaying accordingly